### PR TITLE
release v0.7.05

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

**Comment by andy5995** [comment]
@sexybiggetje if there are any changes you want to see before release, please let me know soon! :) Have to release soon due to #280

File: packaging/Slackbuild/rmw/README.Slackware [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# rmw-0.7.04
```
Comment (andy5995) [inline packaging/Slackbuild/rmw/README.Slackware:1]: This file is just a copy of the main README.md file and gets included when I prepare a Slackbuild package

**Comment by sexybiggetje** [comment]
@andy5995 I'm good I think :)

**Comment by andy5995** [comment]
> @andy5995 I'm good I think :)

@sexybiggetje Just so you know, I haven't asked you to update NL translation because I figured might as well wait until after #281 is fixed. Which isn't going to happen before this release.

**Comment by andy5995** [comment]
Merged from the command line.

**Pull request merged**